components/Article/Article.styled.js

```diff
@@ -19,7 +19,7 @@ const StyledDetailsLink = styled(Link)`
 
 const StyledArticleImageContainer = styled.div`
   position: relative;
-  height: 30vh;
+  height: 35vh;
   background-color: var(--article-card);
   border-radius: var(--border-radius) var(--border-radius) 0 0;
```

Comment:
Was it on purpose that the border-radius is set twice?

Reply:
the first is for the top-left radius and the second is for the top-right radius. the zeros for the lower corners.